zebra-consensus/src/chain/tests.rs

```diff
@@ -320,78 +311,31 @@ async fn verify_fail_add_block_checkpoint_test() -> Result<(), Report> {
 async fn verify_fail_add_block_checkpoint() -> Result<(), Report> {
     zebra_test::init();
 
-    let block =
-        Arc::<Block>::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_GENESIS_BYTES[..])?;
-    let hash: BlockHeaderHash = block.as_ref().into();
-
     let (mut chain_verifier, mut state_service) = verifiers_from_network(Mainnet);
 
-    /// SPANDOC: Make sure the verifier service is ready (1/2)
+    /// SPANDOC: Make sure the block verifier service is ready (1/2)
```

Comment:
Nit: This is actually a ChainVerifier service, not a BlockVerifier service.

So let's be consistent with the other functions:
```suggestion
    /// SPANDOC: Make sure the verifier service is ready (1/2)
```